core/src/main/scala/org/broadinstitute/dsde/rawls/webservice/AdminApiService.scala

```diff
@@ -275,6 +275,13 @@ trait AdminApiService extends HttpService with PerRequestCreator with UserInfoDi
           )
         }
       }
+    } ~
+    path("admin" / "submissions" / "queueStatusByUser") {
```

Comment:
Seems reasonable to me.  Please move this up to group with the other `submissions` endpoints.